src/modules/launcher/Wox.Infrastructure/Image/ImageLoader.cs

```diff
@@ -31,18 +33,20 @@ public static class ImageLoader
         };
 
 
-        public static void Initialize()
+        public static void Initialize(Theme theme)
         {
             _storage = new BinaryStorage<Dictionary<string, int>>("Image");
             _hashGenerator = new ImageHashGenerator();
             ImageCache.SetUsageAsDictionary(_storage.TryLoad(new Dictionary<string, int>()));
 
+            // Todo : Add error and default icon specific to each theme
```

Comment:
Is there anything blocking us from doing that now?

Reply:
We currently don't have theme-specific error/default icons. It will be updated in #4149.